Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1978,6 +1978,20 @@ def create_action(self, machine_id, request_body, overwrite_rate_limit_retry=Fal
                                                overwrite_rate_limit_retry=overwrite_rate_limit_retry)
         return response
 
+    def upload_to_library(self, data, file):
+        cmd_url = 'libraryfiles'
+        response = self.ms_client.http_request(method='POST', url_suffix=cmd_url, data=data, files=file)
+        return response
+
+    def list_library(self):
+        cmd_url = 'libraryfiles'
+        response = self.ms_client.http_request(method='GET', url_suffix=cmd_url)
+        return response
+
+    def delete_from_library(self, filename):
+        cmd_url = f'libraryfiles/{filename}'
+        self.ms_client.http_request(method='DELETE', url_suffix=cmd_url, return_empty_response=True)
```

Comment:
To me it's necessary because if we leave this parameter to False. The response object will automatically try to parse an "empty response" and therefore will crash. We have to pass in the condition `if is_response_empty_and_successful and return_empty_response` in the fonction `http_request`.